examples/webgl_postprocessing_pixel.html

```diff
@@ -0,0 +1,244 @@
+                pixelPass.uniforms.step.value = params.step;
+            }
+
+            function resize(){
+
+                camera.aspect = window.innerWidth / window.innerHeight;
+                camera.updateProjectionMatrix();
+                renderer.setSize( window.innerWidth, window.innerHeight );
+            
+            }
+
+            function init() {
+
+                var container = document.getElementById( 'container' );
+                renderer = new THREE.WebGLRenderer({antialias: true});
+                renderer.shadowMap.enabled = true;
+                renderer.shadowMap.type = THREE.PCSoftShadowMap;
+                renderer.setPixelRatio( window.devicePixelRatio );
+                renderer.setSize( window.innerWidth, window.innerHeight);
+                renderer.setClearColor(0xbfe7ff);
+                container.appendChild( renderer.domElement );
+                
+                camera = new THREE.PerspectiveCamera( 75, window.innerWidth / window.innerHeight, 1, 10000 );
+                camera.position.set(0, 0, 30);
+                controls = new THREE.TrackballControls(camera, renderer.domElement);
+                controls.rotateSpeed = 2.0;
+                controls.panSpeed = 0.8;
+                controls.zoomSpeed = 1.5;
+
+                scene = new THREE.Scene();
+
+                var hemisphereLight = new THREE.HemisphereLight(0xfceafc, 0x000000, .8)
+
+                var shadowLight = new THREE.DirectionalLight(0xffffff, .5);
+
+                shadowLight.position.set(150, 75, 150);
+
+                shadowLight.castShadow = true;
+                shadowLight.shadow.camera.left = -75;
+                shadowLight.shadow.camera.right = 75;
+                shadowLight.shadow.camera.top = 75;
+                shadowLight.shadow.camera.bottom = -75;
+                shadowLight.shadow.camera.near = 1;
+                shadowLight.shadow.camera.far = 1000;
+
+                shadowLight.shadow.mapSize.width = 1024;
+                shadowLight.shadow.mapSize.height = 1024;
+
+                var shadowLight2 = shadowLight.clone();
+                shadowLight2.castShadow = false;
+                shadowLight2.intensity = .2;
+                shadowLight2.position.set(-150, 75, -150);
+
+                var shadowLight3 = shadowLight.clone();
+                shadowLight3.castShadow = false;
+                shadowLight3.intensity = .1;
+                shadowLight3.position.set(0, 125, 0);
+
+                scene.add(hemisphereLight);
+                scene.add(shadowLight);
+                scene.add(shadowLight2);
+                scene.add(shadowLight3);
+
+                geomData.push(new THREE.SphereGeometry(1, 64, 64));
+                geomData.push(new THREE.BoxGeometry(1, 1, 1,));
+                geomData.push(new THREE.ConeGeometry(1, 1, 32));
+                geomData.push(new THREE.TetrahedronGeometry(1));
+                geomData.push(new THREE.TorusKnotGeometry(1, .4));
```

Comment:
This can help with formatting: http://zz85.github.io/mrdoobapproves/

Reply:
...and thank you for your contribution.

Reply:
Removed shadows and formatted the geometry code. Thanks!

Reply:
Your entire PR has formatting problems that need to be fixed.